src/components/Profile/components/MembershipsList/components/MembershipInfoCard/index.jsx

```diff
@@ -15,6 +16,9 @@ const MembershipInfoCard = ({ myProf, membership, onTogglePrivacy }) => {
           className={`gc360_text_link ${
             showPrivate ? styles.private_membership : styles.public_membership
           }`}
+          style={{
+            color: theme.colorSchemes.light.palette.primary[300],
+          }}
```

Comment:
We try to avoid inline styles wherever possible. Can this be updated to be in an scss file instead?